libbpf-rs/src/program.rs

```diff
@@ -36,6 +36,12 @@ impl OpenProgram {
             libbpf_sys::bpf_program__set_ifindex(self.ptr, idx);
         }
     }
+
+    /// Get program section name
+    pub fn get_section_name(&self) -> Result<String> {
+        let section = unsafe { libbpf_sys::bpf_program__section_name(self.ptr) };
+        util::c_ptr_to_string(section)
+    }
 }
```

Comment:
This LGTM. Thanks for adding this

Reply:
let's not use "get_" prefix for getters. Libbpf follows this rule as well.

Reply:
Fixed in next commit